frameworks/CMSIS/LPC1768/Re-ARM/system_LPC17xx.c

```diff
@@ -546,9 +558,15 @@ void SystemInit (void)
 
   LPC_SC->CCLKCFG   = 0x00000002;       /* Setup CPU Clock Divider            */
 
-  LPC_SC->PLL0CFG   = 0x00010018;     // 100MHz
-  LPC_SC->PLL0FEED  = 0xAA;
-  LPC_SC->PLL0FEED  = 0x55;
+  if(can_120MHz()) {
+    LPC_SC->PLL0CFG   = 0x0000000E;     /* configure PLL0                     */
+    LPC_SC->PLL0FEED  = 0xAA;
+    LPC_SC->PLL0FEED  = 0x55;
```

Comment:
We also need to adjust flash access time in FLASHCFG.FLASHTIM as the default is too aggressive for 120MHz. It's currently set to 4 cycles and it should be 5 for 120MHz 1769. [Ref: section 5.4 in the manual](https://www.nxp.com/docs/en/user-guide/UM10360.pdf)